packages/ui/src/i18n/dictionaries/authenticator/ja.ts

```diff
@@ -22,6 +22,7 @@ export const jaDict: AuthenticatorDictionary = {
   'Enter your code': 'コードを入力',
   'Enter your Email': 'メールアドレスを入力',
   'Enter your phone number': '電話番号を入力',
+  'Enter your Username': 'ユーザー名を入力 ',
```

Comment:
I may be wrong but this looks the same as the original string, maybe a copy/paste error?